WordPress/gutenberg
# Gallerys ids are saved as numbers

## Description

 Changed the JSON schema for the gallery blocks to ensure that the ids array is always made with numbers.

## How has this been tested?




- Create a gallery block with images from WP media library
- Switch to code and see that the ids are ints
- Save & Close post
- Reopen post
- Select a tile and move left or right
- Switch to code view and see that the id are still numbers
- Edit the HTML and change the ids to strings
 - Switch to Visual mode and see that all is ok
 - Switch to HTML mode and see if the block was saved with numbers.

## Screenshots 

## Types of changes




Bug fix

## Checklist:
